src/services/shareService.ts

```diff
@@ -24,10 +36,13 @@ export async function getSharings() {
             return { error: 'user not found' };
         }
 
-        const sharings = await SharingModel.find<ISharing>({ user });
+        const sharings = await SharingModel.find<SharingInfo>({ user })
+            .populate('user', 'name')
+            .populate('recipient', 'name')
+            .populate('card', 'image_front');
 
         return {
-            sharings,
+            sharings: JSON.parse(JSON.stringify(sharings)) as SharingInfo[],
```

Comment:
Is it for getting rid of the warning about plain objects?

Reply:
No. There was an stack overflow error when Nextjs try to parse the object to JSON to return to the frontend. This is a work around to deal with that.